docs/resources/vault_secrets_rotating_secret.md

```diff
@@ -0,0 +1,111 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_vault_secrets_rotating_secret Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Vault Secrets rotating secret resource manages a rotating secret configuration.
+---
+
+# hcp_vault_secrets_rotating_secret (Resource)
+
+The Vault Secrets rotating secret resource manages a rotating secret configuration.
+
+## Example Usage
+
+```terraform
+resource "hcp_vault_secrets_rotating_secret" "example_aws" {
+  app_name             = "my-app-1"
+  secret_provider      = "aws"
+  name                 = "my_aws_1"
+  integration_name     = "my-aws-1"
+  rotation_policy_name = "built-in:60-days-2-active"
+  aws_access_keys = {
+    iam_username = "my-iam-username"
+  }
+}
+
+resource "hcp_vault_secrets_rotating_secret" "example_gcp" {
+  app_name             = "my-app-1"
+  secret_provider      = "gcp"
+  name                 = "my_gcp_1"
+  integration_name     = "my-gcp-1"
+  rotation_policy_name = "built-in:60-days-2-active"
+  gcp_service_account_key = {
+    service_account_email = "<name>>@<project>.iam.gserviceaccount.com"
+  }
+}
+
+resource "hcp_vault_secrets_rotating_secret" "example_mongodb_atlas" {
+  app_name             = "my-app-1"
+  secret_provider      = "mongodb_atlas"
+  name                 = "my_mongodb_atlas_1"
+  integration_name     = "my-mongodbatlas-1"
+  rotation_policy_name = "built-in:60-days-2-active"
+  mongodb_atlas_user = {
+    project_id    = "<uuid>>"
+    database_name = "my-cluster-0"
+    roles         = ["readWrite", "read"]
+  }
+}
+
+resource "hcp_vault_secrets_rotating_secret" "example_twilio" {
+  app_name             = "my-app-1"
+  secret_provider      = "twilio"
+  name                 = "my_twilio_1"
+  integration_name     = "my-twilio-1"
+  rotation_policy_name = "built-in:60-days-2-active"
+  twilio_api_key       = {}
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `app_name` (String) Vault Secrets application name that owns the secret.
+- `integration_name` (String) The Vault Secrets integration name with the capability to manage the secret's lifecycle.
+- `name` (String) The Vault Secrets secret name.
+- `rotation_policy_name` (String) Name of the rotation policy that governs the rotation of the secret.
+- `secret_provider` (String) The third party platform the dynamic credentials give access to. One of `aws` or `gcp`.
```

Comment:
nit: could drop "dynamic" here since these fields are shared between both capabilities

Reply:
My bad I missed that comment. I'll fix it in a follow-up.